.github/workflows/build_doc.yml

```diff
@@ -0,0 +1,45 @@
+name: Documentation
+on: [push, pull_request]
+
+jobs:
+  Build_Doc:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone OpenVINO
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          lfs: true
+
+      - name: Install dependencies
+        run: |
+          sudo apt --assume-yes install libusb-1.0-0-dev graphviz texlive
+          python3 -m pip install -r ./inference-engine/ie_bridges/python/requirements.txt
```

Comment:
why do we need `ie_bridges` requirements?

Reply:
It was needed for cmake, probably not needed now

Reply:
can we remove?

Reply:
removed